pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -379,25 +382,18 @@ func isPreferred(a, b granularMode, fungibilityConfig kueue.FlavorFungibility) b
 		return true
 	}
 
-	if !features.Enabled(features.FlavorFungibilityImplicitPreferenceDefault) {
```

Comment:
/hold
We cannot just remove the feature gate. We need to introduce API instead which will be able to express the preferences.